webtool/test_correct.py

```
@@ -1582,4 +1582,69 @@ def fake_correct_file(project, b, gjson, context, verify, force, **kw):
+        (t / f"{b}.json").write_text(json.dumps(raw), encoding="utf-8")
+        (t / f"{b}.raw.txt").write_text(f"Text {b}\n", encoding="utf-8")
+
+    hw_active = 0
+    hw_max = 0
+    hw_lock = threading.Lock()
+
+    ai_active = 0
+    ai_max = 0
+    ai_lock = threading.Lock()
+
+    def fake_diarize(project, only_bases=None):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `fake_diarize`

Add return type annotation: `int`

(ANN202)